overlays/custom-packages/gtklock/default.nix

```diff
@@ -0,0 +1,15 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# This overlay will apply gtklock patches which are merged in master branch
+# https://github.com/jovanlanik/gtklock/commit/d22127f0fd61bbeba8c12378b3c5b46cc3064d63
+# https://github.com/jovanlanik/gtklock/commit/e0e7f6d5ae7667fcc3479b6732046c67275b2f2f
+# TODO: Remove patches, once there new release for gtlk-lock
```

Comment:
please add a link to track these patches, here https://github.com/tiiuae/ghaf/blob/main/overlays/README.md so we can review when to remove them

Reply:
Thanks @brianmcgillion, Updated with patches.